src/liquid/components/ld-tabs/ld-tablist.css

```diff
@@ -0,0 +1,181 @@
+.ld-tablist {
+  --ld-tab-indicator-color: var(--ld-col-bg-g);
+  --ld-tab-btn-scroll-color: var(--ld-col-rblck-default);
+  --ld-tab-btn-scroll-color-active: var(--ld-col-rblck-default);
+  --ld-tab-btn-scroll-bg-color: var(--ld-col-bg-lg);
+  --ld-tab-btn-scroll-bg-color-hover: var(--ld-col-rblck1);
+  --ld-tab-btn-scroll-bg-color-active: var(--ld-col-rblck2);
+  position: relative;
+  display: flex;
+  overflow: hidden;
+  z-index: 0;
+
+  &::after {
+    content: '';
+    position: absolute;
+    inset: auto 0 0 0;
+    height: var(--ld-sp-2);
+    background-color: var(--ld-tab-indicator-color);
+    z-index: 2;
+  }
+
+  .ld-tabs--rounded-all & {
```

Comment:
I thought it would be simpler if the developer had to only add modifiers to one of the components. So I thought the outer one, ld-tabs, would be the one. This also applies to the mode and the size prop by the way. Also, if I added the props to the inner components, I would either have to put them in a separate folder parallel to the ld-tabs folder or I would have to document the props manually. I didn't like both ideas.